src/redux/books/books.js

```diff
@@ -0,0 +1,52 @@
+const ADD = 'book-store/books/ADD ';
+const REMOVE = 'book-store/books/REMOVE';
+
+const initialState = {
+  books: [
+    {
+      id: '1',
+      author: 'Gayle Laakmann McDowell',
+      title: 'Cracking the Coding Interview',
+    },
+    {
+      id: '2',
+      author: 'Robert Greene',
+      title: 'The 48 Laws of Power',
+    },
+  ],
+};
```

Comment:
- [ ] It would be better if you set the initial state with an empty array.

![Capture](https://user-images.githubusercontent.com/64173189/160931505-49fdbc04-a444-449f-b3ee-18864608dac0.JPG)